demosplan/DemosPlanCoreBundle/ResourceConfigBuilder/StatementResourceConfigBuilder.php

```diff
@@ -90,6 +90,8 @@
  * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>, Statement> $segmentDraftList
  * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>,Statement> $availableInternalPhases
  * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>,Statement> $availableExternalPhases
+ * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>,Statement> $phaseStatement
+ * @property-read AttributeConfigBuilderInterface<ClauseFunctionInterface<bool>,Statement> $availablePhases
```

Comment:
Can we rename this properties to: $procedurePhase and $availableProcedurePhases?